Vagrantfile

```diff
@@ -71,6 +71,8 @@ Vagrant.configure(2) do |config|
   # Puppet, Chef, Ansible, Salt, and Docker are also available. Please see the
   # documentation for more information about their specific syntax and use.
   config.vm.provision "shell", inline: <<-SHELL
+    sudo apt-get -y install python-pip
+    sudo pip install ansible==1.9.4
```

Comment:
just a heads up I have a PR up to move this repo to Ansible 2.0 https://github.com/RackHD/on-imagebuilder/pull/23, I can rebase to your changes and then edit this to pull the latest like I do currently (warning build proccess changes to handle issues in the new ansible, I recommend checking out the new readme)